web/locales/en/plugin__netobserv-plugin.json

```diff
@@ -429,12 +429,21 @@
   "Network interface": "Network interface",
   "Specify a network interface.": "Specify a network interface.",
   "Specify a single conversation hash Id.": "Specify a single conversation hash Id.",
+  "Packet drop TCP state": "Packet drop TCP state",
+  "Specify a single TCP state.": "Specify a single TCP state.",
+  "Specify a single TCP state name like:": "Specify a single TCP state name like:",
+  "A _LINUX_TCP_STATES_H number like 1, 2, 3": "A _LINUX_TCP_STATES_H number like 1, 2, 3",
+  "A _LINUX_TCP_STATES_H TCP name like ESTABLISHED, SYN_SENT, SYN_RECV": "A _LINUX_TCP_STATES_H TCP name like ESTABLISHED, SYN_SENT, SYN_RECV",
+  "Packet drop latest cause": "Packet drop latest cause",
+  "Specify a single TCP drop cause.": "Specify a single TCP drop cause.",
+  "Specify a single TCP drop cause like:": "Specify a single TCP drop cause like:",
```

Comment:
this isn't `TCP` specific here and below ?

Reply:
sorry my bad

Reply:
https://github.com/netobserv/network-observability-console-plugin/pull/362/commits/c29eb9f68ffddf4e92f26db9ab8ed802d2788be8